pkg/lightning/backend/tidb/tidb.go

```diff
@@ -309,6 +334,11 @@ type tidbBackend struct {
 	// affecting the cluster too much.
 	maxChunkSize uint64
 	maxChunkRows int
+	// implement stmtCache to improve performance, especially when the downstream is TiDB
```

Comment:
```suggestion
	// implement stmtCache to improve performance
```
we don't support downstream other than TiDB